x/gov/simulation/msgs.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // ContentSimulator defines a function type alias for generating random proposal
 // content.
-type ContentSimulator func(r *rand.Rand) gov.Content
+type ContentSimulator func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simulation.Account) gov.Content
```

Comment:
Why did we add these new parameters?

Reply:
Because we need to read state (check community pool balance & fetch a random account) to simulate community pool spend proposals.